src/changelog/.3.x.x/remove_jakarta_ee.xml

```diff
@@ -15,15 +15,12 @@
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
   -->
-<Configuration status="OFF">
-  <Appenders>
-    <File name="file" fileName="${web:initParam.myapp.logdir}/myapp.log" append="true">
```

Comment:
If we skip the license Git will not confused this with a file rename from `log4j-webvar.xml` to `remove_jakarta_ee.xml`.

Reply:
Agreed. Though ideally this mass remove-the-license-header operation should be performed separate from this PR.